backend/src/packages/orders/order.controller.ts

```diff
@@ -0,0 +1,148 @@
+import { ApiPath } from '~/libs/enums/enums.js';
+import {
+  type ApiHandlerOptions,
+  type ApiHandlerResponse,
+  Controller,
+} from '~/libs/packages/controller/controller.js';
+import { HttpCode } from '~/libs/packages/http/http.js';
+import { type ILogger } from '~/libs/packages/logger/logger.js';
+import { type OrderService } from '~/packages/orders/order.service.js';
+
+import { OrdersApiPath } from './libs/enums/enums.js';
+import {
+  type OrderCreateRequestDto,
+  type OrderEntity,
+  type OrderUpdateRequestDto,
+} from './libs/types/types.js';
+
+type idT = Pick<OrderEntity, 'id'>;
```

Comment:
- type names must be capitalized
- can we have shared type for ID?

Reply:
created shared type for Id